### DISCUSSION TITLE
Add score boost when AR gets below 5 like DT

### DISCUSSION THREAD

**Discussion by youth0530** [ppy/osu#29837] [Ideas]
Today on osu, there's a playstyle called "EZHD" or "EZFL", a mix of EZ and mod that makes hard to see, which makes AR very slow and making very hard to see.
That'll very challenging balance change, but I think it'll make a more flexible play style.

**Comment by cihe13375** [comment]
maybe your suggestion is covered by https://github.com/ppy/osu/discussions/25706 (unluckily the devs seem to don't like the idea?)

**Reply by youth0530** [reply]
Should I close the discussion?
